sample_code/Canvas Example.ipynb

```diff
@@ -0,0 +1,108 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        "host": "https://summitps.instructure.com",
</pre><p>It might be a good idea for the sample to point to the beta instance, in case someone else on Info Team tries running the notebook. Otherwise, we would create users in Prod we don't mean to.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SummitPublicSchools/spswarehouse/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>